### PR TITLE
Bug/cherry pick issues 324 379

### PR DESCRIPTION
**Pull request by afsmythe**
I'm hoping to have the [`release` build](https://vip-specification.readthedocs.io/en/release/index.html) brought up to recent changes to the documentation build process and other fixes presently in place in the `latest` and `vip52` branches of the documentation. This particular merge seems to pull in some older commits from 2016. It's a lot to review, but the changes should only be accounted for by some cherry picking of the commits in PR https://github.com/votinginfoproject/vip-specification/pull/381

### CODE REVIEW

File: vip_spec.xsd [hunk 1]
```diff
@@ -5,7 +5,7 @@
      https://github.com/votinginfoproject/vip-specification
      for history and more infor.
   -->
-<xs:schema attributeFormDefault="unqualified" elementFormDefault="unqualified" version="5.1.2" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="unqualified" version="5.2" xmlns:xs="http://www.w3.org/2001/XMLSchema">
```
Comment (jdmgoogle) [inline vip_spec.xsd]: Are you sure you want to cherrypick the version being 5.2?

**Comment by jdmgoogle** [comment]
One comment. Otherwise looks good.

File: docs/yaml/elements/internationalized_text.yaml [hunk 2]
```diff
@@ -1,34 +1,15 @@
...
-  * :ref:`$$$-party`
-
-  * :ref:`$$$-person`
-
-  * :ref:`$$$-polling-location`
-
-  * :ref:`$$$-source`
-
-  NOTE: Internationalized Text is not currently supported for CSV submissions. 
+description: "``InternationalizedText`` allows for support of multiple languages for\
```
Comment (jdmgoogle) [inline docs/yaml/elements/internationalized_text.yaml]: This seems like noise?

File: vip_spec.xsd (hunk 1 above)
Comment (afsmythe) [reply vip_spec.xsd]: This is fixed.

File: docs/yaml/elements/internationalized_text.yaml (hunk 2 above)
Comment (afsmythe) [reply docs/yaml/elements/internationalized_text.yaml]: This is corrected.

**Pull request merged**